### PR TITLE
update loop in ger

### PR DESCRIPTION
**Pull request by fanatid**
Code that wasn't included in #9

### CODE REVIEW

**Comment by dcousens** [comment]
Why the inversion of logic?
`do while`?

**Comment by calvinmetcalf** [comment]
Yeah while this does remove some duplication I'm not convinced it's actually clearer

File: index.js [hunk 1]
```diff
@@ -9,11 +9,10 @@ function blind (priv) {
...
 function getr (priv) {
   var len = priv.modulus.byteLength()
-  var r = new BN(randomBytes(len))
-  while (r.cmp(priv.modulus) >= 0 || !r.umod(priv.prime1) || !r.umod(priv.prime2)) {
-    r = new BN(randomBytes(len))
+  while (true) {
+    var r = new BN(randomBytes(len))
+    if (r.cmp(priv.modulus) < 0 && r.umod(priv.prime1) && r.umod(priv.prime2)) return r
   }
-  return r
```
Comment (dcousens) [inline index.js:16]: This code,  but leave `r` uninitialized, and `s/while/do` and `s/}/} while (...)`

**Comment by fanatid** [comment]
Updated.

**Comment by fanatid** [comment]
Any chance for new release?

**Comment by dcousens** [comment]
Because of this?  It doesn't change anything does it?

**Comment by fanatid** [comment]
True, it doesn't change result of function.
(but here already 7 commits from last release, but I'm ok if new release doesn't happen)

**Pull request merged**